src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    private final Random random = new Random();
+    private final Color[] colors = Color.values();

```

Comment:
No need to make it class level variable

Reply:
We'll get excessive object creation on each call to the getRandomColor() method if these variables move onto the method level

Reply:
Then be consistent and make this field constant and put there length of colors, not colors themself